src/constants.json

```diff
@@ -33,6 +33,7 @@
     "BID_WON": "bidWon",
     "BIDDER_DONE": "bidderDone",
     "SET_TARGETING": "setTargeting",
+    "BEFORE_REQUEST_BIDS": "beforeRequestBids",
```

Comment:
Should the value here be `beforeRequestBids` or `onBeforeRequestBids`?  The latter was in the description's example setup.

Reply:
Will leave that call to @mkendall07 -- since none of the rest of the events are prefixed with "ON", this seems ok to me as long as the documentation matches.

Reply:
I'm good with the name `beforeRequestBids`.